### ISSUE TITLE
Caps lock dont triggers underscore

### ISSUE THREAD

**Issue by RoniPerson** [godotengine/godot#42774]
**Godot version:**
3.2.3
3.2.2

**OS/device including version:**
Windows 10

**Issue description:**
When using caps lock it dont turns ``-`` in ``_``.

**Steps to reproduce:**
Just open the code editor and try it.

**Comment by vnen** [comment]
Does this work in any other application? AFAIK Caps Lock only work for letters, that's always has been the case for me.

**Comment by Calinou** [comment]
This is likely to be a QWERTY keyboard, since <kbd>Shift + -</kbd> gives you `_` there.

**Comment by name-here** [comment]
I don't know of any application or OS that *does* turn `-` into `_` when caps lock is on.  I don't see any reason Godot should have this feature—it would probably annoy more people than it helped.

**Comment by KoBeWi** [comment]
According to Wikipedia, some keyboards indeed treat Caps Lock as Shift:

> In most cases, the status of the Caps Lock key only changes the meaning of the alphabet keys, not that of any other key. Microsoft Windows enforces this behavior only when a keyboard layout for a Latin-based script is active, e.g. the "English (United States)" layout but not the "Persian" layout. However, on certain non-QWERTY keyboard layouts, such as the French AZERTY and the German QWERTZ, Caps Lock still behaves like a traditional Shift lock, i.e., the keyboard behaves as if the Shift key is held down, causing the keyboard to input the alternative values of the keys; example the "5" key generates a "%" when Caps lock is pressed. This is not true for the layout "German (IBM)".
~https://en.wikipedia.org/wiki/Caps_Lock

If it works correctly in other applications but not in Godot then it's probably a bug.

**Comment by RoniPerson** [comment]
I use one of the keyboard layouts that are mentioned for behaving like shift. At the time I opened this it used to work this way in other applications but at the time beeing it dosn't anymore. Therefore I assume that Windows now enforces it. In retrospective I can not be sure if the issue was caused by a bug or a strange driver setup on my side. Therefore I will close the issue. Even if it was a bug it would not have any priority.
